integration/query_test.go

```diff
@@ -327,12 +394,23 @@ func TestQueryMaxTimeMSErrors(t *testing.T) {
 	} {
 		name, tc := name, tc
 		t.Run(name, func(t *testing.T) {
+			if tc.skip != "" {
+				t.Skip(tc.skip)
+			}
+
 			t.Parallel()
 
+			require.NotNil(t, tc.command, "command must not be nil")
+			require.NotNil(t, tc.err, "err must not be nil")
+
 			var actual bson.D
 			err := collection.Database().RunCommand(ctx, tc.command).Decode(&actual)
-			require.Error(t, err)
-			AssertEqualAltError(t, *tc.err, tc.altMessage, err)
+			if tc.altMessage != "" {
```

Comment:
In principle, we can move this logic right to `AssertEqualAltCommandError` - https://github.com/FerretDB/FerretDB/pull/2752/files#diff-001e940d5c8698ead39d9f150b62f80c03958fa26aca39adb3e402bac00b9e51, and then `AssertEqualAltCommandError` can be used without any other conditions. Otherwise, this `if tc.altMessage != ""` will be repeated in every single test.

(And if we do it, I guess if we do it, we should do the same for `AssertEqualAltWriteError` too).

Reply:
Great let's do that! Thanks for the lint :pray: